install/installer/pkg/components/server/configmap_test.go

```diff
@@ -30,6 +31,7 @@ func TestConfigMap(t *testing.T) {
 	expectation := Expectation{
 		EnableLocalApp:                    true,
 		DisableDynamicAuthProviderLogin:   true,
+		RunDbDeleter:                      false,
```

Comment:
The default should be true. This is the default I assume?

Reply:
This is for the test code. I set it to `false` here because it's *not* the default, ie test that setting it explicitly has the desired effect.

 We should probably have multiple test cases that set each of these boolean flags differently.

Reply:
Thx, got it!